packages/contentful--app-scripts/package.json

```diff
@@ -27,7 +27,7 @@
     "prettier": "prettier **/*.js --write --ignore-path .gitignore",
     "lint": "eslint ./lib",
     "lint:fix": "npm run lint -- --fix",
-    "test": "mocha ./lib/**/*.test.js ./lib/*.test.js ./utils/**/*.test.js",
+    "test": "mocha \"./{,!(node_modules)/**/}*.test.js\" --exit",
```

Comment:
Why this change?

Reply:
previous one was weird because every time you add a new folder you'd need to remember to pass the folder in here. I opted to just run everything with the `.test.js` suffix and also exit the test runner as previously we weren't even running them in the first place 🤦🏾

Reply:
The trick (I think) for mocha is to use quotes. Running `mocha "path/**/file.js"` is different to `mocha path/**/file.js`.

This also does the job and is maybe easier to understand as it doesn't have negation: 
```
mocha \"./{lib,utils}/**/*.test.js\" --exit
```

Reply:
That's exactly the problem I was trying to avoid - forgetting add new folders here because I'm pretty sure it will happen again 😅 . I'd be in favour of moving `utils` folder into the `lib` but that's probably a discussion for another PR

Reply:
+1 for only having one source folder.
I guess I have the same discussion in another PR atm: https://github.com/contentful/create-contentful-app/pull/1634#discussion_r1302874023